src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnTestUtil.kt

```diff
@@ -15,5 +23,45 @@ object TilsynBarnTestUtil {
         beregningsresultat = beregningsresultat,
     )
 
+    val vedtakBeregningsresultat = VedtaksdataBeregningsresultat(
+        perioder = listOf(
+            Beregningsresultat(
+                dagsats = BigDecimal.TEN,
+                månedsbeløp = 1000,
+                grunnlag = Beregningsgrunnlag(
+                    måned = YearMonth.now(),
+                    makssats = 1000,
+                    stønadsperioderGrunnlag = emptyList(),
+                    utgifter = emptyList(),
+                    utgifterTotal = 2000,
+                    antallBarn = 1,
+                ),
+                beløpsperioder = listOf(
+                    Beløpsperiode(dato = osloDateNow(), beløp = 1000, målgruppe = MålgruppeType.AAP),
+                    Beløpsperiode(dato = osloDateNow().plusMonths(1), beløp = 2000, målgruppe = MålgruppeType.AAP),
```

Comment:
Tror vi har blitt enige om å ikke bruke `osloDateNow()` men heller bruke `LocalDate.now()` 🤓

Reply:
Sant sant, kopierte bare over en funksjon fra en annen fil så den ble tilgjengelig for flere tester. Kan endre den